docker-compose.yaml

```diff
@@ -1,5 +1,20 @@
 version: '3.9'
 services:
+  website:
+    build:
+      context: .
+      dockerfile: ./website/Dockerfile
```

Comment:
does it mean to update the website we will need to remove the local image and then rebuild it ?

Reply:
Actually we can just run docker-compose up --build